src/components/case-details-card.js

```diff
@@ -561,17 +565,38 @@ export default function CaseDetailsCard({ ID }) {
                 ) : null}
               </StyledActionsDiv>
               <StyledDiv className="secondary-background theme-background" style={{ display: "inherit" }}>
-                {Number(dispute.period) < "3" && !votesData.voted ? (
-                  <Button
-                    disabled={!votesData.canVote}
-                    ghost={votesData.canVote}
-                    id={0}
-                    onClick={onVoteClick}
-                    size="large"
-                  >
-                    Refuse to Arbitrate
-                  </Button>
-                ) : null}
+                <div>
+                  {Number(dispute.period) < "3" && !votesData.voted ? (
+                    <Button
+                      disabled={!votesData.canVote}
+                      ghost={votesData.canVote}
+                      id={0}
+                      onClick={onVoteClick}
+                      size="large"
+                    >
+                      Refuse to Arbitrate
+                    </Button>
+                  ) : null}
+                </div>
+                {metaEvidence.metaEvidenceJSON.rulingOptions &&
+                  metaEvidence.metaEvidenceJSON.rulingOptions.reserved &&
+                  metaEvidence.metaEvidenceJSON.rulingOptions.reserved[UINT_MAX] && (
+                    <div style={{ marginTop: "32px" }}>
+                      {Number(dispute.period) < "3" && !votesData.voted ? (
+                        <Button
+                          disabled={!votesData.canVote}
+                          ghost={votesData.canVote}
+                          id={UINT_MAX}
+                          onClick={onVoteClick}
+                          size="large"
+                        >
+                          {metaEvidence.metaEvidenceJSON.rulingOptions &&
+                            metaEvidence.metaEvidenceJSON.rulingOptions.reserved &&
+                            metaEvidence.metaEvidenceJSON.rulingOptions.reserved[UINT_MAX]}
+                        </Button>
+                      ) : null}
+                    </div>
+                  )}
```

Comment:
This basically solves the problem of Realitio, but not the general one.
Since we added the reserved rulings to the standard, this should handle the general case, iterating over `rulingOptions.reserved` and displaying all of them.